backend/fms_core/models/sample.py

```diff
@@ -240,11 +240,11 @@ def source_depleted(self) -> bool:
         return self.extracted_from.depleted if self.extracted_from else None
 
     @property
-    def extracted_from(self) -> ["Sample"]:
+    def extracted_from(self) -> "Sample":
```

Comment:
I think at some point you were talking about referring to multiple samples (array)? Did you change your mind on that?

Reply:
Nope i just removed that so it would match the first() in the function code... if (when) we change first() for all() we'll add back the [ ].

Reply:
I am surprised that we can change the return type without much consequences on the rest of our app.. As long as it does not have side effects...